app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/accounts/homepage/HomePageController.scala

```diff
@@ -87,7 +87,7 @@ class HomePageController @Inject() (
             .fold(
               _ =>
                 triage.routes.CommonTriageQuestionsController
-                  .howManyProperties(),
+                  .furtherReturnHelp(),
```

Comment:
this should only happen if it's a further return - you can tell by looking into `sentReturns` in `subscribed` and seeing if it's empty or not

Reply:
if it's not a further return it should go to the original page